Basic/Invaders/Assets/Scripts/EnemyBullet.cs

```diff
@@ -24,6 +23,7 @@ private void Start()
 
     private void Update()
     {
+        if (!NetworkManager.Singleton.IsServer) return;
```

Comment:
Why are we no longer returning here? This will update the bullets position on the clients even though the bullets contain a `NetworkTransform` right?

Reply:
I don't understand if we are not a Server we return since we don't want the client to update the bullets since they are NetworkTransform. So we are actually returning.